index.js

```diff
@@ -1267,20 +1309,9 @@
   //. The environment is only significant if the type contains
   //. [type variables][].
   //.
-  //. One may define a more restrictive type in terms of a more general one:
-  //.
-  //. ```javascript
-  //. //    NonNegativeInteger :: Type
-  //. const NonNegativeInteger = $.NullaryType(
-  //.   'my-package/NonNegativeInteger',
-  //.   'http://example.com/my-package#NonNegativeInteger',
-  //.   x => $.test([], $.Integer, x) && x >= 0
-  //. );
-  //. ```
-  //.
-  //. Using types as predicates is useful in other contexts too. One could,
-  //. for example, define a [record type][] for each endpoint of a REST API
-  //. and validate the bodies of incoming POST requests against these types.
+  //. Using types as predicates is can be useful. One could, for example,
```

Comment:
`s/ is//`

I think *powerful* better captures my thought. How about the following?

_Using types as predicates is powerful._